jbt-it/neomdb
# 456 client trainee application admin page

How to test:
- make sure you have applications in your database otherwise create new ones
- go to the db and set eingelade and aufnehmen for some to true
- run the tests and see if all tests pass
- go to http://localhost:3000/traineebewerbungen
- check if the trainees are displayed correctly
- check if changing the rating is working
- check if the statistics are correct
- check if you can select applicants to invite and for the ones who already are invited the note "eingeladen" shows and they are not selectable
- check if the button "Bewerber einladen" works, if you selected some applicants there should be an indication that they are "eingeladen"
- check if you can only select applicants to admit who have the aufnehmen value in the db and for the ones who are already admitted the note "aufgenommen" shows and they are not selectable
- check if the button "Trainees aufnehmen" works, if you selected some applicants there should be an indication that they are "aufgenommen", you can also check in the "Gesamtübersicht" if you can find the new trainee there
